app/assets/stylesheets/views/_landing_page/hero.scss

```diff
@@ -34,7 +36,7 @@ $large-desktop-height: 610px;
 .app-b-hero__image {
   display: block;
 
-  @media (min-width: 1281px) {
+  @include govuk-media-query($from: desktop) {
```

Comment:
This has changed the behaviour slightly, just need to check. 

Previously the hero on desktop was a fixed height and didn't scale, so image content could be lost left/right. That was fine until it passed the width of the image at 1280, and a gap appeared.

To fix this, we added a new breakpoint above desktop (> 1280) to make the image absolute and truly full width, but retaining a fixed height, with image content potentially being lost off the bottom.

I think the requirement was only to set the image to scale at an aspect ratio for tablet and mobile, with the behaviour on desktop being acceptable. Shall we catch up about this?